api/swagger.yaml

```diff
@@ -4226,6 +4226,7 @@ paths:
             Filters to process on the prune list, encoded as JSON (a `map[string][]string`).
 
             Available filters:
+            - `until=<string>` Prune containers created until this timestamp.
```

Comment:
Perhaps we should use `<timestamp>` here as well, and explain the format? Does the API take the `Go` format, or is this a numeric (UNIX) timestamp?